.prettierrc

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Já navrhuju Prettier vůbec nepoužívat, nastavit ho tak, aby fungovat s ESLintem je strašná otrava a z mojí skušenosti to přínáší jen víc problémů. ESLint je dostatečně schopný sám o sobe.

Reply:
no já mám ale prettier nějak ve vscode, a nastavený na "", když zmačknu save tak tak mi tam skočí všude "", tak bez něj se zase zbláznim s přepisováním '', to se mi ručně nechce dělat a nevim jak vypnu teda to automatzické formátování co mám  nastavené, na jiné projekty to tak zase potřebuju...

Reply:
Přepokládám, že to nastavení Prettieru máš přímo v settings VS Code. V tom případě můžeš v root toho projektu udělat složku `.vscode` a do ní soubor `settings.json`. Tam můžeš to použití Prettieru vypnout přímo pro tenhle projekt a žádný jiný.